cuebot/src/main/java/com/imageworks/spcue/dao/postgres/DispatcherDaoJdbc.java

```diff
@@ -353,6 +355,24 @@ public boolean findUnderProcedJob(JobInterface excludeJob, VirtualProc proc) {
          }
     }
 
+    @Override
+    public boolean findHigherPriorityJob(JobDetail baseJob, VirtualProc proc) {
```

Comment:
`findHigherPriorityJob` makes me think the method will return a job. Maybe `higherPriorityJobExists`?

Reply:
done.